example/android/build.gradle

```diff
@@ -33,5 +33,6 @@ allprojects {
         google()
         jcenter()
         maven { url 'https://www.jitpack.io' }
+        mavenCentral()
```

Comment:
Like to know why is this needed

Reply:
If I put `mavenCentral` in the package's build.gradle, it won't find the facebook-android-sdk:9. I had to put it in rout project level

Reply:
Tanks for the explanation.

Just take a look to the `README.md`, this might be missing for a long time.